orchestrator/db/models.py

```diff
@@ -81,6 +83,20 @@ def process_result_value(self, value: datetime | None, dialect: Dialect) -> date
         return value.astimezone(timezone.utc) if value else value
 
 
+class InputStateTable(BaseModel):
+    __tablename__ = "input_state"
```

Comment:
```suggestion
    __tablename__ = "input_states"
```

(almost) All other table names are plural as well